File: 3600-find-the-k-th-character-in-string-game-i/solution.java
class Solution {
    public char kthCharacter(int k) {
        int n = k-1;
        int ans = 0;
        for(int i = 0; i<32; i++){
            if((n&(1<<i)) != 0){
                ans++;
            }
        }
        return (char)('a' + ans);
    }
}

File: 3600-find-the-k-th-character-in-string-game-i/solution.py
class Solution:
    def kthCharacter(self, k: int) -> str:
        word = 'a'
        while len(word) <k:
            new_word = ''
            for i in word:
                if i == 'z':
                    ch = 'a'
                else:
                    ch = chr(ord(i) + 1)
                new_word = new_word + ch
            word = word + new_word
        print(word)
        return word[k-1]
        
